src/components/AllJobLogsView/AllJobLogsView.js

```diff
@@ -51,48 +51,50 @@ const initialQuery = '?qindex=hrID&sort=-completedDate';
 const defaultSorting = '-completedDate';
 const defaultSortingSearch = '?sort=-completedDate';
 
-const buildJobsQuery = makeQueryBuilder(
-  `status=(${JOB_LOGS_STATUS_QUERY_VALUE})`,
-  query => `query=${query}`,
-  'sortby completedDate/sort.descending',
-  {
-    completedDate: buildDateTimeRangeQuery.bind(null, ['completedDate']),
-    startedDate: buildDateTimeRangeQuery.bind(null, ['startedDate']),
-    status: query => {
-      switch (true) {
-        case query === JOB_EXECUTION_STATUSES.FAIL:
-          return jobStatusFailString;
-        case Array.isArray(query):
-          return `status=(${query.map(v => (
-            v === JOB_EXECUTION_STATUSES.COMPLETED
-              ? JOB_LOGS_STATUS_QUERY_VALUE
-              : `"${v}"`)).join(' or ')})`;
-        default:
-          return `status=${query}`;
-      }
+const buildJobsQuery = (timezone) => {
```

Comment:
is it possible to avoid this change?

Reply:
let me check, if it possible to get access to timezone in other way

Reply:
I haven't found another way to get access to stripes.timzone